### PR TITLE
Remove oc setup for devfile push docker target

### PR DESCRIPTION
**Pull request by amitkrout**
**What type of PR is this?**
/kind cleanup
**What does does this PR do / why we need it**:
Removing unnecessary ```oc cluster up```
https://github.com/openshift/odo/pull/2921#issuecomment-617240185
**Which issue(s) this PR fixes**:

Fixes NA

**How to test changes / Special notes to the reviewer**:
Test job ```docker devfile push command integration tests``` should pass in travis CI

### CODE REVIEW

**Comment by amitkrout** [comment]
ping @johnmcollier @kadel

**Comment by girishramnani** [comment]
@amitkrout 
```
Running odo with args [odo url create jqcbc --host 1.2.3.4.com]
[odo]  ✗  invalid configuration: no configuration has been provided
Running odo with args [odo url create tqblw --host 1.2.3.4.com]
[odo]  ✗  invalid configuration: no configuration has been provided
``` 
seem like genuine failures

**Comment by amitkrout** [comment]
> @amitkrout
> 
> ```
> Running odo with args [odo url create jqcbc --host 1.2.3.4.com]
> [odo]  ✗  invalid configuration: no configuration has been provided
> Running odo with args [odo url create tqblw --host 1.2.3.4.com]
> [odo]  ✗  invalid configuration: no configuration has been provided
> ```
> 
> seem like genuine failures

Yes, looking into it

**Comment by amitkrout** [comment]
> @amitkrout
> 
> ```
> Running odo with args [odo url create jqcbc --host 1.2.3.4.com]
> [odo]  ✗  invalid configuration: no configuration has been provided
> Running odo with args [odo url create tqblw --host 1.2.3.4.com]
> [odo]  ✗  invalid configuration: no configuration has been provided
> ```
> 
> seem like genuine failures

Fixed via - https://github.com/openshift/odo/commit/1fd4c8ef3a4311eba58c3ab177256eb407632ea1

File: .travis.yml [hunk 1]
```diff
@@ -122,6 +122,20 @@ jobs:
         - travis_wait make test-cmd-app
         - travis_wait make test-cmd-push
         - travis_wait make test-cmd-project
+        - odo logout
+
+    - <<: *base-test
+      stage: test
+      name: "devfile catalog, create, push, delete and docker devfile url command integration tests"
+      script:
+        - ./scripts/oc-cluster.sh
```
Comment (girishramnani) [inline .travis.yml:131]: Shouldn’t we be removing this? That is oc cluster setup for docker and kube?

Comment (amitkrout) [reply .travis.yml:131]: I did not get the context of what you expecting/asking, Are you looking a separate job for this ? can you please explain bit more ?

Actually my thought was, without creating a separate job how i can optimize/rearrange the test file for docker push target test with the actual kube config file. For example ```test-cmd-docker-devfile-url``` needs kube config file for certain scenario. So what i did i just move the test location to a place where it needs a fresh kube config. 

However i kept the rest of test file such as ```test-cmd-docker-devfile-push```, ```test-cmd-docker-devfile-catalog``` and ```test-cmd-docker-devfile-delete``` in a separate job without a Kube environment i mean no ```oc cluster up```.

Optimization it brings
- Use the kube file without creating a separate job.
- Remove oc cluster up for test that does not need Kube environment. (Time decreased by one oc cluster up)

Comment (amitkrout) [reply .travis.yml:131]: @girishramnani As per our slack discussion i will update the comment message. Thanks for the feedback

Comment (amitkrout) [reply .travis.yml:135]: Updated
ping @girishramnani

**Pull request merged**